depends/hosts/darwin.mk

```diff
@@ -71,6 +71,10 @@ $(foreach TOOL,$(cctools_TOOLS),$(eval darwin_$(TOOL) = $$(build_prefix)/bin/$$(
 #
 #         Adds the desired paths from the SDK
 #
+#     -platform_version
```

Comment:
Does this obsolete/interact with mlinker-version at all?

Reply:
From having another look now, Clang still uses `-mlinker-version`, in it's decision making about whether to use `-platform_version` (by default), which it does if the linker version is > 520. If we are passing `-platform_version` ourselves, then maybe we can drop `-mlinker-version`, but I would have to look again, to see if there are other side effects, and this is probably ok to leave for now, and revisit when we do the lld switch, as I assume more flags are going to change there as well.